script.js

```diff
@@ -1,62 +1,55 @@
-const imgArr=["./num0.jpg", "./num1.png", "./num2.png", "./num3.png", "./num4.png", "./num5.png", "./num6.jpeg"
-, "./num7.png", "./num8.png", "./num9.jpg", "./num10.jpg", "./num11.png", "./num12.png", "./num13.png"
-, "./num14.png", "./num15.png", "./num16.jpeg", "./num17.png", "./num18.png", "./num19.jpg"];

```

Comment:
This can be dynamically generated in a loop, or even better, using array functions (`from`, `keys`, `map`, etc.)
See [this list of possible array functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)
from the documentation for ideas on how you can manipulate arrays
to reach a result you desire.